dev-client/src/translations/en.json

```diff
@@ -56,15 +56,10 @@
     "form": {
       "name_min_length_error": "Site name must be at least {{min}} characters",
       "name_max_length_error": "Site name must be at most {{max}} characters",
-      "coords_parse_error": {
-        "COORDS_PARSE": "Coordinates must be comma-separated decimal numbers representing a latitude, longitude pair",
-        "LATITUDE_PARSE": "Latitude must be a decimal number",
-        "LATITUDE_MIN": "Latitude must be at least {{LATITUDE_MIN}}",
-        "LATITUDE_MAX": "Latitude must be at most {{LATITUDE_MAX}}",
-        "LONGITUDE_PARSE": "Longitude must be a decimal number",
-        "LONGITUDE_MIN": "Longitude must be at least {{LONGITUDE_MIN}}",
-        "LONGITUDE_MAX": "Longitude must be at most {{LONGITUDE_MAX}}"
-      }
+      "min_latitude_error": "Latitude must be at least {{LATITUDE_MIN}}",
```

Comment:
Issue: You need to either change `LATITUDE_MIN` to `min` here to match the field specified in siteValidationSchema.ts, or vice versa. 

Same with max, and with longitude min/max.

Especially since we don't have automated tests right now, please make sure to manually confirm all added functionality is working as expected as much as possible.